src/Grid.js

```diff
@@ -3,21 +3,20 @@ import Button from './button';
 
 import './App.css';
 
+const Step = ({ step = [], index, toggleButton }) => (
```

Comment:
Because this component doesn't rely on any state or anything we can just define it as a plain function, where the props are passed as the first argument, which I've immediately destructured here.